src/modules/navigator/navigator_main.cpp

```diff
@@ -960,6 +960,13 @@ void Navigator::check_traffic()
 			continue;
 		}
 
+
+		//ADSB Transmitter 14 = UAV
+		if (tr.emitter_type == 14) {
+			horizontal_separation = 10;
+			vertical_separation = 10;
```

Comment:
Should these perhaps be parameters, instead of hard-coded? (This question is also valid for the separation distance for manned vehicles, which is currently hard-coded as 500 meters)